### PR TITLE
Common PR for all previously opened PRs

### PR DESCRIPTION
**Pull request by mayank7**
Added more steps, corrected the sourceUrl and fixed typos
IA Page: https://duck.co/ia/view/mtg_phases_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by moollaza** [comment]
@mayank7  thanks a lot for doing this!

File: share/goodie/cheat_sheets/json/filezilla.json [hunk 1]
```diff
@@ -36,9 +36,21 @@
...
             "key":"Enter",
             "val":"Transfers the currently selected item if it is a file or expands it if it is a directory"
+          }, {
+            "key":"[Ctrl] [PgUp]",
+            "val":"Switches to next tab"
+          }, {
+            "key":"[Ctrl] [PgDn]",
+            "val":"Switches to previous tab"
+          }, {
+            "key":"[↑]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/filezilla.json]: Single keys should not be wrapped in `[]` -- they're only used to signify individual keys

File: share/goodie/cheat_sheets/json/github-flavoured-markdown.json [hunk 2]
```diff
@@ -49,6 +50,19 @@
...
 			"val": "You can use a subset of HTML within your READMEs, issues, and pull requests"
+		}],
+		"Styling Text": [{
+			"key": "**Bold Text**",
+			"val": "Bold"
+		},{
+			"key": "*Italic Text*",
+			"val": "Italic"
+		},{
+			"key": "~~Mistaken Text~~",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: Strikethrough is defined above -- I think we should remove it as a standalone section and keep it here

File: share/goodie/cheat_sheets/json/gta-vice-city.json [hunk 3]
```diff
@@ -109,6 +109,30 @@
...
+      {
+        "val": "Makes you smoke a cigarette",
+        "key": "CERTAINDEATH"
+      },
+      {
+        "val": "Only the wheels of a car are visible",
+        "key": "WHEELSAREALLINEED"
+      },
+      {
+        "val": "Peds Carry Guns",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: Let's spell out `Pedestrians` fully

File: share/goodie/cheat_sheets/json/filezilla.json (hunk 1 above)
Comment (mayank7) [inline share/goodie/cheat_sheets/json/filezilla.json]: @moollaza Just me give me some time to merge all my PRs here, then I will go through all your review msg & will do the corrections

Comment (mayank7) [inline share/goodie/cheat_sheets/json/filezilla.json]: @moollaza Removed square brackets for single keys

File: share/goodie/cheat_sheets/json/github-flavoured-markdown.json (hunk 2 above)
Comment (mayank7) [inline share/goodie/cheat_sheets/json/github-flavoured-markdown.json]: @moollaza I removed Inline html section and put its contents in Strikethrough

File: share/goodie/cheat_sheets/json/gta-vice-city.json (hunk 3 above)
Comment (mayank7) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: @moollaza Changed Peds to Pedestrians

File: share/goodie/cheat_sheets/json/cryptography.json [hunk 4]
```diff
@@ -26,6 +26,9 @@
             "val": "A cryptographic algorithm, i.e. a mathematical function used for encryption and decryption.",
             "key": "Cipher"
         }, {
+            "val": "It is a method of encrypting data bit-by-bit, as opposed to encoding a contiguous chunk of data all at once like a block cipher.",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/cryptography.json]: I would say "piece-by-piece" unless the explicit concept of 1 and 0 being encrypted is deliberately implied

File: share/goodie/cheat_sheets/json/elementary.json [hunk 5]
```diff
@@ -90,11 +94,11 @@
             },
             {
                 "val": "Will take the screentshot of entire screen",
-                "key": "prscr"
+                "key": "Prscr"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/elementary.json]: I think this should be `prtsc`?

File: share/goodie/cheat_sheets/json/filezilla.json (hunk 1 above)
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/filezilla.json]: @mayank7 these are still wrapped?

File: share/goodie/cheat_sheets/json/groovy.json [hunk 6]
```diff
@@ -176,6 +177,36 @@
                 "key": "while (<expr>) {...}",
                 "val": "Basic while loop"
             }
+        ],
+        "Escape characters": [
+            {
+                "key": "\t",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/groovy.json]: These need to be literal `\` so this key should be `"\\t"`

File: share/goodie/cheat_sheets/json/gta-vice-city.json (hunk 3 above)
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/gta-vice-city.json]: @mayank7 this doesn't appear to be the case?

**Comment by mayank7** [comment]
@moollaza @mintsoft The branch corresponding to this PR in my system has got very badly screwed up, so I have created two new PR's namely "Common PR" and "Common_2" for my open PR's and also incorporating the changes you people suggested.
So, I am closing this branch to avoid redundancy.

**Pull request closed without merging**